template/function/serverless.yml

```diff
@@ -1,21 +1,29 @@
-service: app
+service: TPL_SERVICE_NAME
 
 provider:
-    name: aws
-    region: us-east-1
-    runtime: provided.al2
+  # general settings
+  name: aws
+  stage: dev
+  region: TPL_REGION
+  profile: default
+  deploymentMethod: direct
+  # global function settings (apply to each declared function)
+  runtime: provided.al2
+  memorySize: 1024
+  timeout: 6
+  logRetentionInDays: forever
+  layers:
+    - ${bref:layer.php-TPL_PHP_VERSION}
 
 plugins:
-    - ./vendor/bref/bref
+  - ./vendor/bref/bref
 
 functions:
-    hello:
-        handler: index.php
-        description: ''
-        layers:
-            - ${bref:layer.php-PHP_VERSION}
+  hello:
+    description: 'Deployed with Bref TPL_BREF_VERSION for PHP TPL_PHP_VERSION_PRETTY'
```

Comment:
I don't think this is a good description of what the function does. Only the user knows what to put in here no?